config/application.properties

```diff
@@ -27,6 +27,11 @@ spring.datasource.url=jdbc:postgresql://localhost:5433/yaci_indexer?currentSchem
 spring.datasource.username=user
 spring.datasource.password=
 
+# # Use defaults, but allow Docker env override
```

Comment:
@nelsonksh Thanks.

I think with the current approach, we always need to rebuild the image whenever there's an update to the database info in application.properties.

Let me merge this for now. 

One possible enhancement could be to mount the application.properties file, allowing it to be updated externally without rebuilding the entire image.